docs/getting-started/preview.md

```diff
@@ -0,0 +1,79 @@
+# W4H-Preview
+## Introduction
+This document is for the preview of W4H toolkit. W4H Toolkit is an ecosystem that covers most of the common features needed when using wearables in a clinical study or or trial. As studies can have very different requirements, W4H toolkit is designed to be flexible and incrementally adoptable. This toolkit provides the following components:
+
+
+**Dashboard** : An intuitive interface that offers data input forms and visual analytics tools.
```

Comment:
Just say what it is not stuff that ChatGPT suggests!

Look at the ICDE demo video/paper (Samrit knows ask in Slack). You want to write something like this:

- Streamlit dashboard featuring:
    - Import data
    - ...  
- Database supporting GeoMTS
- Notebooks:
    - Compliance notebooks in pandas/PySpark